src/shell/lisa_shell

```diff
@@ -373,6 +373,69 @@ echo
 
 
 ################################################################################
+# LISA Workloads utility functions
+################################################################################
+
+function lisa-wltest-init {
+export LISA_VENV="$LISA_HOME/tools/wltests/wa3.venv"
+export LISA_WAUTO="$LISA_HOME/tools/wltests/wlauto"
+export WA_USER_DIRECTORY="$LISA_HOME/tools/wltests/.wlauto"
```

Comment:
Again I would call this dir `.workload_automation` (note underscore) to make it more similar to the default